core/camel-core/src/test/java/org/apache/camel/builder/FluentProducerTemplateTest.java

```diff
@@ -39,19 +40,19 @@
 public class FluentProducerTemplateTest extends ContextTestSupport {
 
     @Test
-    public void testNoEndpoint() throws Exception {
+    public void testNoEndpoint() {
         FluentProducerTemplate fluent = context.createFluentProducerTemplate();
 
         try {
             fluent.withBody("Hello World").send();
-            fail("Should have thrown exception");
+            Assertions.assertDoesNotThrow(() -> fail("Should have thrown exception"));
```

Comment:
This code is wrong as the code was expected to throw an exception, and hence it will execute the .. catch block.

Reply:
Yeah, now I see. Thanks, I'll fix it